Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
+#	my %headers;
+#        warn $p{'path'};
+	unless ($p{'path'}) {
+	    die "No path specified";
+	}
+#	$headers{'Content-Type'} = 'application/json';
+#	if (defined($p{'headers'})) {
+#	    %headers = (%headers, %{$p{'headers'}});
+#	}
+#	my $headers = HTTP::Headers->new( %headers);
+    
+#	my $client = LWP::UserAgent->new(default_headers => $headers);
+	my $client = LWP::UserAgent->new();
+	$client->timeout(5);
+	$client->ssl_opts(verify_hostname => 0);
```

Comment:
Dunno.  This was just copy/paste from the go_request method.  It could be an artifact from when we didn't have the wildcard cert on ci?  Anyway, it probably makes sense to remove this option.